src/components/container/NotificationList/NotificationList.tsx

```diff
@@ -65,7 +66,7 @@ export default function (props: NotificationListProps) {
 		return () => {
 			MyDataHelps.off("applicationDidBecomeVisible", initialize);
 		}
-	}, []);
+	}, [props.notificationType]);
```

Comment:
Just double checking - this fixes the fact that in the View Builder PR, changing the Notification Type doesn't change the preview, right?

Reply:
Correct, it also allows toggling the notification type in the default story